### PR TITLE
New Port: Septuple

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Septuple

A precision platformer about the septuple jump paper! Created by NNB. 

Instructions: Download septuple.x86_64 from https://nnbnh.itch.io/septuple and place it into gamedata folder

CFW Tests
[x] AmberELEC
[x] ArkOS
[x] JELOS / ROCKNIX
[x] muOS 

Architecture
[X] aarch64
[] Panfrost (Mainline) 

Resolution Tests
Test all major resolutions:
[x] 480x320 (Optional)
[x] 640x480
[x] 720x720 (RGB30)
[x] Higher resolutions (e.g., 1280x720

### CODE REVIEW

File: ports/septuple/README.md
Comment (Cebion) [inline ports/septuple/README.md:1]: Readme does not have enough infomraiton.

Should have proper Credits in with the url to the gamepage.

Controls section

See https://portmaster.games/packaging.html#readme-md

File: ports/septuple/Septuple.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+$ESUDO mkdir -p "$godot_dir"
+$ESUDO umount "$godot_file" || true
+$ESUDO mount "$godot_file" "$godot_dir"
+PATH="$godot_dir:$PATH"
+
+export FRT_NO_EXIT_SHORTCUTS=FRT_NO_EXIT_SHORTCUTS
+
+$ESUDO chmod 666 /dev/uinput
+$GPTOKEYB "$runtime" -c "./septuple.gptk" &
+SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig" "$runtime" $GODOT_OPTS --main-pack "gamedata/septuple.x86_64" -sw 2>&1 | $ESUDO tee -a ./log.txt
```
Comment (Cebion) [inline ports/septuple/Septuple.sh:58]: Wrong logging method.
> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1

File: ports/septuple/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Septuple.sh</path>
+    <name>Septuple</name>
+    <desc>A precision platformer about the septuple jump paper! https://nnbnh.itch.io/septuple</desc>
+    <releasedate>20221029T000000</releasedate>
+    <developer>NNB</developer>
+    <publisher>NNB</publisher>
+    <genre>Platformer</genre>
+    <image>./septuble/screenshot.png</image>
```
Comment (Cebion) [inline ports/septuple/gameinfo.xml:11]: septuple or septuble?

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]
👍

**Pull request merged**